tests/e2e/vsan_stretched_cluster.go

```diff
@@ -277,12 +297,14 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 		err = waitForAllNodes2BeReady(ctx, client)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
+		time.Sleep(5 * time.Minute)
 		// Check if csi pods are running fine after site failure
 		err = fpod.WaitForPodsRunningReady(ctx, client, csiNs, int32(csipods.Size()), 0, pollTimeout)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 		ginkgo.By("Verifying volume lifecycle actions works fine")
 		volumeLifecycleActions(ctx, client, namespace, sc)
```

Comment:
see, if we can return some kind of error from this util. If there is some issue in the code, it will fail in the caller methods, lets return the output to the parent method

Reply:
This is an existing method, changing this would require to modify in multiple places where ever it is getting used.